draft-beky-httpbis-metadata.md

```diff
@@ -65,9 +65,11 @@ Both HTTP/2 and HTTP/3 specifications allow the protocol to be extended, see
 
 This document defines a new frame type: METADATA.
 
-The payload of a METADATA frame is a metadata block, which is an encoded list
-of key-value pairs.  Each key and value is a sequence of bytes with no
-restriction on the allowed values.
+The payload of a sequence of METADATA frames is a metadata block, which is an
+encoded list of key-value pairs.  Each key and value is a sequence of bytes with
+no restriction on the allowed values.
+
+A connection or stream _MAY_ carry any number of metadata blocks.
```

Comment:
This might be very nitpicky, but I would use some verb other than "carry". Maybe transmit, send, or transfer?